### PR TITLE
Update navigation-timing-api.md

### PR DESCRIPTION
**Pull request by nem035**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/javascript/browser-apis/measuring-performance/navigation-timing-api.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/browser-apis/measuring-performance/navigation-timing-api.md?ref=4da092c6fed9a34d420c31ccc2fabad353581f7f)

File: javascript/browser-apis/measuring-performance/navigation-timing-api.md [hunk 1]
```diff
@@ -42,14 +29,21 @@ The Navigation Timing API provides timing information about the page load proces
 The API records the time at key points in the page load process and stores each event as a propriety of the `window.performance.timing` object.
```
Comment (Stefan-Stojanovic) [inline javascript/browser-apis/measuring-performance/navigation-timing-api.md]:
```suggestion
The API records the time at key points in the page load process and stores each event as a property of the `window.performance.timing` object.
```

The revision question (Line 86) also needs the same change, but I can't add a comment on that part of the code.

File: javascript/browser-apis/measuring-performance/navigation-timing-api.md [hunk 2]
```diff
@@ -58,12 +52,13 @@ window.addEventListener("load", function() {
...
-var redirectCount =
-       navigation.redirectCount;
+
+```js
+let navigation =
+  window.performance.navigation;
+let navType = navigation.type;
+let redirectCount =
+  navigation.redirectCount;
 // number of redirects to current page
```
Comment (Stefan-Stojanovic) [inline javascript/browser-apis/measuring-performance/navigation-timing-api.md]:
```suggestion
// number of redirects to the current page
```

File: javascript/browser-apis/measuring-performance/navigation-timing-api.md [hunk 3]
```diff
@@ -1,33 +1,20 @@
...
 aspects:
   - new
   - workout
   - deep
   - obscura
 
 links:
-
   - '[www.sitepoint.com - apis](http://www.sitepoint.com/10-html5-apis-worth-looking/){website}'
   - '[www.sitepoint.com - profiling](http://www.sitepoint.com/profiling-page-loads-with-the-navigation-timing-api/){website}'
```
Comment (kapnobatai137) [inline javascript/browser-apis/measuring-performance/navigation-timing-api.md]: are these going to be too long?
```suggestion
  - '[10 HTML5 APIs Worth Looking Into](http://www.sitepoint.com/10-html5-apis-worth-looking/){website}'
  - '[Profiling Page Loads with the Navigation Timing API](http://www.sitepoint.com/profiling-page-loads-with-the-navigation-timing-api/){website}'
```

**Pull request merged**